.github/workflows/release.yml

```diff
@@ -0,0 +1,158 @@
+name: Create release
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set_variables.outputs.version }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
+
+    - name: Install Hatch
+      run: pipx install hatch
+
+    - name: Set variables
+      id: set_variables
+      run: |
+        tag=$(git for-each-ref refs/tags --sort=-creatordate --format='%(refname:strip=2)' --count=1)
+        cp README.md plugin/
+        cd plugin/
+        echo "::group::Variables"
+        cat << EOF | tee -a "$GITHUB_OUTPUT"
+        tag=${tag}
+        version=v$(hatch project metadata | jq -r .version)
```

Comment:
Do you want to keep the git tag naming style in your repo(like `0.2.1`) or do you want the one with prefix v(like `v0.2.1`)?

Reply:
Let's keep it without the prefix

Reply:
done in 0696592 (not yet tested)